Source/SlimDetours/Trampoline.c

```diff
@@ -400,24 +400,29 @@ detour_is_region_empty(
     return TRUE;
 }
 
+static
 VOID
-detour_free_unused_trampoline_regions(VOID)
+detour_free_region(
+    _In_ PDETOUR_REGION* ppRegionBase,

```

Comment:
```suggestion
    _Out_ PDETOUR_REGION* ppRegionBase,
```
Seems like an output parameter.

Reply:
Right, fixed, thanks.